ideascube/templatetags/ideascube_tags.py

```diff
@@ -130,3 +132,37 @@ def paginate(request, **kwargs):
     for key, value in kwargs.items():
         get[key] = value
     return '?{}'.format(get.urlencode())
+
+
+@register.simple_tag(takes_context=True)
+def add_qs(context, **kwargs):
+    req = template.resolve_variable('request', context)
+    params = req.GET.copy()
+    for key, value in kwargs.items():
+        if key.endswith('s'):
```

Comment:
We use add_qs and remove_qs in django template.
But there is no easy way to create a list from a single element in django template.

However, we do not use this functionality (we provide a single element) so we may simply remove the functionality and assume is always a single element.
What do you think ?